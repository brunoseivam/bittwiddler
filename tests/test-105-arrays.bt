main {
    var a:int8[] = [1, 2, 3, 4, 5];
    var l = len(a);

    emit("The array a has {l} elements\n");
}
