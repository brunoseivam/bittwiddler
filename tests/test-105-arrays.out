The array a has 5 elements
